cmd/policy-assistant/pkg/matcher/explain.go

```diff
@@ -141,3 +140,14 @@ func PortMatcherTableLines(pm PortMatcher, kind PolicyKind) []string {
 		panic(errors.Errorf("invalid PortMatcher type %T", port))
 	}
 }
+
+func priorityTableLine(e Effect) string {
+	if e.PolicyKind == NetworkPolicyV1 {
+		return "All peers allowed"
+	} else if e.PolicyKind == AdminNetworkPolicy {
+		return fmt.Sprintf("%s (%s): %s (pri=%d)", e.PolicyKind, e.Source, e.Verdict, e.Priority)
+	} else {
+		return fmt.Sprintf("%s (%s): %s", e.PolicyKind, e.Source, e.Verdict)
```

Comment:
nit: could you either add check/panic for BANP or just a clarifying comment that this is BANP in case we add more PolicyKind's later?